go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/sylabs/singularity/v4
 
-go 1.20
+go 1.21.0
+
+toolchain go1.21.8
```

Comment:
Don't think we want a `toolchain` directive here?

```suggestion
```

Reply:
Thanks... must have re-appeared from picking the fixes on main in the wrong order.